Added unsupported animated stickers information (#98)
sinch/docs: docs/whatsapp/whatsapp-http-rest/whatsapp-message.md

2. Stickers must be exactly 512x512 pixels.
3. Each sticker must be less than 100 KB.

> Please note that that WhatsApp doesn't support animated stickers.

Reviewer: "that that", one "that" too many. Better with "does not" (no contraction) in more formal text such as this.

> Please note that that WhatsApp doesn't yet support animated stickers.

Reviewer: ```suggestion
Please note that animated stickers are currently not supported.
```
